content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -0,0 +1,109 @@
+---
+reviewers:
+- jpbetz
+- cheftako
+title: "Controller Manager Leader Migration"
+linkTitle: "Controller Manager Leader Migration"
+content_type: task
+---
+
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.21" >}}
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
+
+## Background
+
+Kubernetes project is in the process of migrating the cloud specific code from the core Kubernetes tree to external packages and removing them from the `kube-controller-manager`. Once the extraction has been completed, existing users running older versions of Kubernetes need a process to migrate their existing clusters to use the new `cloud-controller-manager` component with minimal risk.
+
+Leader Migration provides a mechanism in which HA clusters can safely migrate "cloud specific" controllers between the `kube-controller-manager` and the `cloud-controller-manager` via a shared resource lock between the two components.
+
+This guide walks you through the process of upgrading the control plane from `kube-controller-manager` with built-in cloud provider to running both `kube-controller-manager` and `cloud-controller-manager`.
```

Comment:
Point out that these are the manual steps required to do the migration, but that cluster administration tools and cloud provider specific documentation might offer more details or an automated workflow?

Reply:
clarified this is the manual process.